spec/models/rook_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe Rook, type: :model do
+  describe "#valid_move?" do
+    let(:rook) { Rook.create(row: 0, col: 1) }
+
+    it "should allow legal vertical moves" do
```

Comment:
You are right, technically this test passes. But as a general rule, a test should cover all the common scenarios. Determining how many scenarios to test, however, is a matter of opinion. For example, should we test `rook.valid_move?(0, 1)` ?

Since this requested change is not absolutely necessary, I'll retract it.